.changelog/1127.txt

```diff
@@ -0,0 +1,63 @@
+```release-note:bug
+`data-source/pingone_agreement`: Fixed case insensitive lookup by name
```

Comment:
Would "fixed case sensitive" be more accurate? This almost sounds like the problem was that it was case insensitive before, which is backwards.

Maybe just reword to something more affirmative like "Corrected lookup by X to be case insensitive"